### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by xSevithx** [xNocken/replay-reader#26]
Looks like the new update may have broken parsing?

**Comment by MineBlo** [comment]
![image](https://github.com/user-attachments/assets/6f1a9b30-6974-4701-a11a-75099c675339)
Is this the error you got? I just downloaded and while I could successfully use the 2 sample replays but any replays from the current build seems to not work

**Comment by xSevithx** [comment]
yea I think the format on replays changed

**Comment by kwkevinlin** [comment]
I am seeing the same error as well. `Too much read was expected 15866`. Previous replays are working but not the new ones.

**Comment by kwkevinlin** [comment]
Hi @xNocken, would you know how to fix this issue? Looks like replay parsing capability is breaking for all new replay files

**Comment by theauthenticator** [comment]
![Image](https://github.com/user-attachments/assets/47f22a0d-2bff-4a84-84ea-f08623f3b676)

Was there any update on this? Trying to read Chapter 6 replays give this error:

**Comment by kwkevinlin** [comment]
It is still broken unfortunately.

**Comment by theauthenticator** [comment]
yeah i moved onto the C# version at [(https://github.com/Shiqan/FortniteReplayDecompressor)],it has good basic data but im currently working out how to get the Damage, Distance and Materials Farmed etc etc.

**Comment by kwkevinlin** [comment]
@theauthenticator Were you able to get it working? How is it? It doesn't look like there are many alternatives.
